WordPress/src/androidTest/java/org/wordpress/android/e2e/ContactUsTests.java

```diff
@@ -0,0 +1,84 @@
+package org.wordpress.android.e2e;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.wordpress.android.R;
+import org.wordpress.android.e2e.flows.LoginFlow;
+import org.wordpress.android.e2e.pages.ContactSupportScreen;
+import org.wordpress.android.support.BaseTest;
+
+import static org.wordpress.android.BuildConfig.E2E_WP_COM_USER_EMAIL;
+import static org.wordpress.android.support.WPSupportUtils.pressBackUntilElementIsDisplayed;
+
+public class ContactUsTests extends BaseTest {
+    @Before
+    public void setUp() {
+        logoutIfNecessary();
+    }
+
+    @After
+    public void tearDown() {
+        pressBackUntilElementIsDisplayed(R.id.continue_with_wpcom_button);
+    }
+
+    @Test
+    public void sendButtonEnabledWhenTextIsEntered() {
+        try {
+            new LoginFlow()
+                .chooseContinueWithWpCom()
+                .tapHelp()
+                .assertHelpAndSupportScreenLoaded()
+                .openContactUs()
+                .assertContactSupportScreenLoaded()
+                .assertSendButtonDisabled()
+                .setMessageText("Hello")
+                .assertSendButtonEnabled()
+                .setMessageText("")
+                .assertSendButtonDisabled();
+        } finally {
+            new ContactSupportScreen().goBackAndDeleteUnsentMessageIfNeeded();
+        }
```

Comment:
A kind of local teardown, in order to not slow down `helpCanBeOpenedWhileEnteringEmail` and `helpCanBeOpenedWhileEnteringPassword` which do not get to `ContactSupport` screen, so there's no possibility of unsent message.